app/shared/containers/Global/Button/Elevate.js

```diff
@@ -0,0 +1,137 @@
+// @flow
+import React, { Component } from 'react';
+import { bindActionCreators } from 'redux';
+import { connect } from 'react-redux';
+import { translate } from 'react-i18next';
+import compose from 'lodash/fp/compose';
+import { Button, Form, Header, Icon, Input, Message, Modal, Segment } from 'semantic-ui-react';
+
+import * as WalletActions from '../../../actions/wallet';
+
+class GlobalButtonElevate extends Component<Props> {
```

Comment:
Just a thought. Why not put this under Global/Modal rather than Global/Button ?

Reply:
Hrm, yeah it could be abstracted further to have it's modal in it's own component, but it was a "button wrapper" (always required another button to be passed), so that's why I put it here to start. We could definitely break this out though!